engine/lite/embeddedDeploymentManager/src/test/scala/pl/touk/nussknacker/streaming/embedded/StreamingEmbeddedDeploymentManagerRestartTest.scala

```diff
@@ -47,6 +47,8 @@ class StreamingEmbeddedDeploymentManagerRestartTest extends BaseStreamingEmbedde
       jobStatuses.map(_.status) shouldBe List(SimpleStateStatus.Restarting)
     }
 
+    // We have to recreate kafka server because after shutdown it is unusable anymore
+    kafkaServer.recreateKafkaServer()
     kafkaServer.kafkaServer.startup()
```

Comment:
`kafkaServer.kafkaServer` looks weird. Can we rename the field to `underlying` or sth like that?

Reply:
I made it more explicit but slightly otherwise